synapse/lib/layer.py

```diff
@@ -3480,13 +3479,15 @@ def _editNodeDel(self, buid, form, edit, sode, meta):
 
         self.buidcache.pop(buid, None)
 
+        sode.pop('valu', None)
         self.mayDelBuid(buid, sode)
+        self.setSodeDirty(buid, sode, form)
```

Comment:
We'll need to call `setSodeDirty` before `mayDelBuid` (or we could make mayDelBuid return a bool and skip setSodeDirty if it was deleted)

Reply:
Excellent idea, thanks! Implemented return bool for `mayDelSode()` :+1: